spec.md

```diff
@@ -2606,6 +2742,24 @@ enumerate(["zero", "one", "two"])               # [(0, "zero"), (1, "one"), (2,
 enumerate(["one", "two"], 1)                    # [(1, "one"), (2, "two")]
 ```
 
+### float
+
+`float(x)` interprets its argument as a floating-point number.
+
+If x is a `float`, the result is x.
+
+If x is an `int`, the result is the floating-point value nearest x.
+The call fails if x is too large to represent as a `float`.
+
+If x is a `bool`, the result is `1.0` for `True` and `0.0` for `False`.
+
+If x is a string, the string is interpreted as a floating-point literal.
+The function also recognizes the names `Inf` (or `Infinity`) and `NaN` of
+the IEEE 754 non-finite values, optionally preceded by a `+` or `-` sign.
+Letter case is not significant.
+
+With no argument, `float()` returns `0.0`.
```

Comment:
This is probably the correct behavior, but `int()` and `str()` are both errors (but `bool()` returns False). Fix by making those also return their zero type?

Reply:
I have a separate TODO item to deal with that.